pkg/virt-launcher/virtwrap/network/podinterface_test.go

```diff
@@ -89,6 +89,7 @@ var _ = Describe("Pod Network", func() {
 		Handler = mockNetwork
 		testMac := "12:34:56:78:9A:BC"
 		updateTestMac := "AF:B3:1F:78:2A:CA"
+		mtu = 1410
```

Comment:
Do you understand how the test passed before this change?

Reply:
I do; in https://github.com/kubevirt/kubevirt/pull/4506/files/11f4614bf1d0263816c4c0c305df4aa1d07cd511..a7c77597389937bb9108080b1d6390f1a744d2f5#diff-7ca01cbb567927ef22a4ae7dfb82dcc71ccd5b3d5d3980eba72d5389bfc15d0cL142 it was using the literal integer 1410, instead of the variable.

I chose to be a good little boy-scout and make this mess of a unit test file a little bit better.

Reply:
Oh right, the number was hard-coded. Thanks!